scripts/binaryninja/manticore_viz/plugin.json

```diff
@@ -0,0 +1,15 @@
+{
+	"plugin": {
+		"name": "ManticoreTrace",
+		"type": ["ui"],
+		"api": "python2",
+		"description": "Manticore Trace Visualization.",
+		"longdescription": "This plugin visualizes Manticore execution traces.\n",
+		"license": {
```

Comment:
i'm not sure what license we should have on this exactly, can we omit this for now?